src/Symfony/Component/Intl/Data/Generator/LanguageDataGenerator.php

```diff
@@ -192,7 +192,7 @@ private function generateAlpha3Codes(array $languageCodes, ArrayAccessibleResour
         }));
 
         foreach ($metadataBundle['alias']['language'] as $alias => $data) {
-            if (3 === \strlen($alias) && 'overlong' === $data['reason']) {
+            if (3 === \strlen($alias) && \in_array($data['reason'], ['overlong', 'bibliographic'], true)) {
```

Comment:
im not sure they should be included in the full list: https://en.wikipedia.org/wiki/ISO_639-2#B_and_T_codes

> In general the T codes are favored;

Reply:
reverted